keyboards/cassette42/keymaps/default/keymap.c

```diff
@@ -0,0 +1,183 @@
+/* Copyright 2019 monksoffunk
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#ifdef RGBLIGHT_ENABLE
+// Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+rgblight_config_t        RGB_current_config;
+#endif
```

Comment:
```suggestion
```

Reply:
This section is requird for WRTROM and RGBRST
https://github.com/monksoffunk/qmk_firmware/blob/e17d5cbfb4d6c04c4fa838cb7b9b05daa13748a9/keyboards/cassette42/keymaps/default/keymap.c#L50-L64

Reply:
They aren't though.  There are functions for every setting. 
https://docs.qmk.fm/#/feature_rgblight?id=query